### ISSUE TITLE
To-do

### ISSUE THREAD

**Issue by SheepTester** [SheepTester/platformre#17]
- [ ] fallen leaves compostenize
- [ ] charcoal as fuel
- [ ] back walls for underground (such as mining stone ▶️  stone back)
- [x] https://formspree.io/ for submitting levels
- [ ] reset slots only after assembling (so that the scroll in assembler doesn't reset)
- [x] have platformre/ instead of platformre/randomlevelgen.html
- [x] have platformre/make/ instead of platformre/levelmaker.html
- [x] lavatosolid turns invs lava ➡️  invs solid and dangerous solid ➡️  solid lava

**Comment by sysdl132** [comment]
![图片](https://user-images.githubusercontent.com/54708046/75145887-dadc2d80-5734-11ea-872e-02e9dee6bc80.png)
Something went wrong.
The correct URL should is:

https://sheeptester.github.io/platformre/gotit.html

not:

https://sheeptester.github.io/platformre **/platformre** /gotit.html

**Comment by SheepTester** [comment]
@sysdl132 Fixed!

**Comment by SheepTester** [comment]
Also, next time please create a separate issue; thanks!
